Resources/doc/mailchimp.md

```diff
@@ -27,8 +27,23 @@ add the following config to `app/config/config.yml`
 ```yml
 sulu_form:
     mailchimp_api_key: %parameter_recommended_for_mailchimp_api_key%
+    mailchimp_subscribe_status: %parameter_recommended_for_mailchimp_subscribe_status%
 ```
 
+## Subscribe Status
+
+- subscribed
```

Comment:
Describe here how to configure pending status all other statuses makes here no sense:

To change the subscribe status from **subscribed** to **pending** you need to configure the following:

```yml
sulu_form:
    mailchimp_subscribe_status: pending
```

Describe not what it does as here it should just be link to the mailchimp documentation: https://developer.mailchimp.com/documentation/mailchimp/guides/manage-subscribers-with-the-mailchimp-api/#check-subscription-status